recipe/meta.yaml

```diff
@@ -10,7 +10,7 @@ source:
   sha256: ddbcda194f49e0cf0663fa8131cb9d7a3b876d14dea0047d3c5fdfaf20adbb40
 
 build:
-  number: 200
+  number: 201
```

Comment:
We can probably skip the build number bump. The main thing it would do for us is show these links on Anaconda.org, which is nice, but maybe not crucial. Then again feel free to correct me if that was part of your interest here.

Reply:
thanks @jakirkham. No, I had no specific interest, I was just trying do the right thing, but I understand it's not needed.